web/packages/teleport/src/config.ts

```diff
@@ -360,7 +361,13 @@ const cfg = {
       export: '/v1/webapi/sites/:clusterId/integrations/:name/ca',
     },
 
+    // TODO(kimlisa): move integrationsPath into integration: {...}
     integrationsPath: '/v1/webapi/sites/:clusterId/integrations/:name?',
+    integration: {
+      deleteWithCleanup:
```

Comment:
You seem to alternate between `Cleanup` and `CleanUp`. I would pick one and stick with it.